inc/namespace.php

```diff
@@ -80,7 +80,22 @@ function merge_config_settings( array $config, array $overrides ) : array {
 			// Merge module settings together.
 			case 'modules':
 				foreach ( $value as $module => $settings ) {
-					$config[ $key ][ $module ] = array_merge( $config[ $key ][ $module ] ?? [], $settings );
+					$config_settings = $config[ $key ][ $module ] ?? [];
```

Comment:
Let's move this back to the `array_merge()` call, we don't need to use this variable anymore.